src/app/tide/spectrum_collection.cc

```diff
@@ -37,8 +37,8 @@ Spectrum::Spectrum(const pb::Spectrum& spec) {
   double m_z_denom = spec.peak_m_z_denominator();
   double intensity_denom = spec.peak_intensity_denominator();
   for (int i = 0; i < size; ++i) {
-    CHECK(spec.peak_m_z(i) > 0);
-    total += spec.peak_m_z(i); // deltas of m/z are stored
+    // CHECK(spec.peak_m_z(i) > 0);
+    total = spec.peak_m_z(i); // deltas of m/z are stored
```

Comment:
We changed how the peaks are stored in the spectrum records file. we now store the actual peak_mz in the protobuff, so that we do not need to order the peaks. it is not neccessary, and it save time.

Reply:
I updated the comment to reflect this.